recipes/pytest-tornasync/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pytest-tornasync" %}
+{% set version = "0.6.0.post1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5fe41d6d98d4650898c082a69338d25879cadd0e55805c9a02bc3eb7212a2ea1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools
+  run:
+    - pytest >=3.0
+    - python >=3.5
+    - tornado >=5.0
+
+test:
+  imports:
+    - pytest_tornasync
+
+about:
+  home: https://github.com/eukaryote/pytest-tornasync
+  license: MIT
+  license_family: MIT
+  license_file: {{ RECIPE_DIR }}/LICENSE
```

Comment:
You don't need this Jinja variable anymore. Please, remove it in the feedstock

Reply:
Let it just ``LICENSE`` and it is fine

Reply:
And please create an issue in pytest-tornasync to package the license as well

Reply:
> And please create an issue in pytest-tornasync to package the license as well

The issue is already there. Thanks

Reply:
> You don't need this Jinja variable anymore. Please, remove it in the feedstock

Nice!

Reply:
> > You don't need this Jinja variable anymore. Please, remove it in the feedstock
> 
> Nice!

I will change it on feedstock! Thanks